buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -424,6 +424,9 @@
     },
     "PB_CleanAgent": {
       "value": "true"
+    },
+    "PB_SymbolsBuildId": {
+      "value": "DotNet-CoreFx-Windows"
```

Comment:
There's no reason we can't wait and change this later, but it's something to figure out before splitting off a release branch.